src/Code.gs

```diff
@@ -161,6 +164,18 @@ const ChatGPTApp = (function () {
     .setDescription("Fetch the viewable content of a web page. HTML tags will be stripped, returning a text-only version.")
     .addParameter("url", "string", "The URL to fetch.");
 
+  let imageDescriptionFunction = new FunctionObject()
+    .setName("getImageDescription")
+    .setDescription("To retrieve the description of an image.")
+    .addParameter("imageUrl", "string", "The URL of the image.")
+    .addParameter("fidelity", "string", "Either \"low\" or \"high\", default \"low\" is sufficient in most cases.");
+
+  let getSpreadsheetContentFunction = new FunctionObject()
+    .setName("getSpreadsheetContent")
+    .setDescription("To access the content of a Spreadsheet")
+    .addParameter("spreadsheetId", "string", "The ID of the spreadsheet")
+    .addParameter("sheetName", "string", `The name of a specific sheet to access. Use "all" to access every sheet`);
```

Comment:
maybe sheetName could be optional and if not provided we return values of all sheets